infra/base-images/base-runner/bad_build_check

```diff
@@ -131,15 +144,53 @@ function check_startup_crash {
 function check_asan_build {
   local FUZZER=$1
   local ASAN_CALLS=$2
-  local MSAN_CALLS=$3
```

Comment:
Nit: It looks like the order of these params were changed to be in alphabetical order, not sure this is worth doing (in fact, I think it is harmful when we adopt this convention in places where it isn't helpful and where obeying simply makes code more difficult to modify).

Reply:
I definitely share your point that sometimes it becomes too annoying, but it feels like we've used to follow that convention in CF / OSS-Fuzz code, so I've just done the same here as well.